core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/BaseAwsJsonProtocolFactory.java

```diff
@@ -242,6 +242,18 @@ public final SubclassT protocolVersion(String protocolVersion) {
             return getSubclass();
         }
 
+        /**
+         * ContentType  of the client (By default it is application/x-amz-json-<JSON version>).
```

Comment:
This default will change soon in the future (like, the next few months - there's a SEP on the way), and we'll probably forget to update this comment. Should we link to where the default is defined instead?

Reply:
Added link to  code where we set the default value ```AWS_JSON = new DefaultJsonContentTypeResolver("application/x-amz-json-");```